hydrus/itemhelpers.py

```diff
@@ -155,3 +170,98 @@ def items_delete_check_support(id_, class_type, path):
         error = e.get_HTTP()
         return set_response_headers(jsonify(error.generate()),
                                     status_code=error.code)
+
+
+def itemsCollection_get_support(collection, class_path, path, search_params):
+            try:
+                # Get collection details from the database
+                if get_pagination():
+                    # Get paginated response
+                    response = crud.get_collection(
+                        get_api_name(), collection.class_.title, session=get_session(),
+                        paginate=True, path=path, page_size=get_page_size(),
+                        search_params=search_params)
+                else:
+                    # Get whole collection
+                    response = crud.get_collection(
+                        get_api_name(), collection.class_.title, session=get_session(),
+                        paginate=False, path=path, search_params=search_params)
+
+                response["search"] = add_iri_template(path=class_path,
+                                                      API_NAME=get_api_name())
+
+                return set_response_headers(jsonify(hydrafy(response, path=path)))
+
+            except (ClassNotFound, PageNotFound, InvalidSearchParameter, OffsetOutOfRange) as e:
+                error = e.get_HTTP()
+                return set_response_headers(jsonify(error.generate()), status_code=error.code)
+
+
+def itemsCollection_post_support(object_, link_props, path):
+    try:
```

Comment:
Please add doctrings to all your functions.

Reply:
Doctrings should be in a format similar to https://github.com/HTTP-APIs/hydrus/blob/37af5f3be07b5b21e133c40e3586f58859f38094/hydrus/helpers.py